src/mockDebug.ts

```diff
@@ -356,7 +356,7 @@ class ProphetDebugSession extends LoggingDebugSession {
 									presentationHint = {
 										kind: 'class'
 									};
-								} else if (member.type === 'Object' && member.value !== 'null') {
+								} else if (member.type === 'Object' || member.type === 'object' && member.value !== 'null') {
```

Comment:
I would prefer to add parentheses around `or` condition, so sentence would be much readable

Reply:
@SqrTT -- Updated with parenthesis.